apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -406,7 +425,7 @@ export const UnconnectedJsDebugger = Radium(React.createClass({
         >
           <span
             style={[
-              this.state.windowWidth <= 1275 && !this.state.watchersHidden && styles.hidden,
+              this.state.consoleWidth <= 345 && styles.hidden,
```

Comment:
If the intent is to start hidden, would it make more sense to have the default be 0?